magprime/automated_emails.py

```diff
@@ -135,7 +135,7 @@ def __init__(self, event):
 
 StopsEmailFixture(
     'Last Chance to enter your MAGFest staff shirt preferences', 'second_shirt.html',
-    lambda a: not a.shirt_info_marked,
+    lambda a: a.gets_staff_shirt and not a.shirt_info_marked,
```

Comment:
This email won't send out anyway because of how I changed shirt_info_marked, but still... best to actually have the right conditions.